### PR TITLE
Blog App: validations and model specs :rocket: 

### PR DESCRIPTION
**Pull request by Juanse7793**
In this PR I fulfilled the following requirements

## Project requirements :heavy_check_mark: 

- Add the following validations:
  - For the User model: :nerd_face: 
    - Name must not be blank. :heavy_check_mark: 
    - PostsCounter must be an integer greater than or equal to zero. :heavy_check_mark: 
  - For the Post model: :100: 
    - Title must not be blank. :heavy_check_mark: 
    - Title must not exceed 250 characters. :heavy_check_mark: 
    - CommentsCounter must be an integer greater than or equal to zero. :heavy_check_mark: 
    - LikesCounter must be an integer greater than or equal to zero. :heavy_check_mark: 
- Add unit specs for all of your models' methods and validations. :exploding_head:

### CODE REVIEW

File: spec/models/user_spec.rb [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    @user = User.new(name: 'John Doe', photo: 'https://example.com/photo.jpg', bio: 'hola', posts_counter: 0)
+    @user.save
+  end
+
+  it 'name should be present' do
+    @user.name = nil
+    expect(@user).to_not be_valid
+  end
+
+  it 'photo always valid' do
```
Comment (RolandM99) [inline spec/models/user_spec.rb:14]: - [ ] Your Model specs should have at least one **Validation** test. Kindly, make sure that you implement it. Take a look at this example below: 
```ruby
describe 'validations' do
  it { should validate_presence_of(:name) }
end
```

**Review by RolandM99 — CHANGES_REQUESTED** [review]
Hello @Juanse7793, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### To highlight 💯 
- Good commit message ✔️ 
- No Linter errors ✔️ 
- Follow Github flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Rolandm99 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/models/user_spec.rb (hunk 1 above)
Comment (Juanse7793) [reply spec/models/user_spec.rb:14]: HI, please can you show me that what you are suggesting works?

I mean I have 24 validations made and you said that there is no one?

let me explain to you how I have done this:
Screenshot from 2022-07-07 13-19-32

in Line 10 in my model, I'm passing PRESENCE: TRUE

so there is no need to modify it to pass validates_presence_of: 

and then in my tests in line 11 as you can see I'm validating presence of 'name' so if 'name' is nil the '@user should not be valid'

Screenshot from 2022-07-07 13-28-53

**Review by Deepakdanger — APPROVED** [review]
Hi @Juanse7793, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [x] Here, the previous reviewer meant to test directly the presence of the `name` in the user model but here you have taken the example that validates that the `user.name` should be present. Almost the same thing, but in the future kindly use the previous reviewer's described method to validate any model attributes.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**